Portfolio/index.html

```diff
@@ -1,209 +1,236 @@
-        </div>
-        </p>
+            <h2><strong>Front End Developer</strong></h2>
 
-        <h4>
-            <div>Name:John Doe</div>
-            <div>Age:24</div>
-            <div>Experience: 2years</div>
-        </h4>
+        </div>
+        <div class="Get_konow_me">
+
+            <div class="containers">
+                <h3>Get To Know me</h3>
+
+                <p><strong>I'm a seasoned front-end web developer with a passion for crafting exceptional digital
+                        experiences.
+                        With over 2 years of hands-on experience, i bring a deep proficiency in HTML, CSS, JavaScript,
+                        TypeScript, and a keen mastery of modern frameworks like React.js, React Native, Svelte.js. I
+                        specialize in building responsive, interactive, and visually appealing user interfaces.
+                        Additionally, my proficiency in Firebase ensures secure and efficient backend integration,
+                        making me
+                        a versatile and dynamic developer ready to tackle any project</strong>
+                </p>
+                <h4>
+                    <div><strong> Name:John Doe</strong> </div>
+                    <div><strong>Age:24</strong> </div>
+                    <div><strong> Experience: 2years</strong> </div>
+                </h4>
+            </div>
         </div>
     </section>
 
     <!-- skills -->
-    <section class="skills" id="skills">
-
-        <h1>SKILLS</h1>
 
-        <h4> <img src="/html.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">HTML</div>
-        </h4>
+    <section class="myskills">
 
-        <h4> <img src="/css.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">CSS</div>
-        </h4>
+        <h1>Skills</h1>
+        <div class="containerss">
 
-        <h4> <img src="/js.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">JavaScript</div>
-        </h4>
 
-        <h4> <img src="/ts.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">TypeScript</div>
-        </h4>
+            <div class="image-container">
+                <img src="/html.icon.png" alt="Sample Image" width="30px" height="30px">
+                <h2>HTML</h2>
+            </div>
 
-        <h4> <img src="/svelte.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">Svelte.js</div>
-        </h4>
+            <div class="image-container">
+                <img src="/css.icon.png" alt="Sample Image" width="30px" height="30px">
+                <h2>CSS</h2>
+            </div>
+            <div class="image-container">
+                <img src="/js.icon.png" alt="Sample Image" width="30px" height="30px">
+                <h2>JavaScript</h2>
+            </div>

```

Comment:
The 'max-width' property value should include a unit (e.g., px) in the .containers class.